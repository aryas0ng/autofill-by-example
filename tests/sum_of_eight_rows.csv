11,85
15,79
2,70
5,77
3,75
2,
4,
43,
5,
6,
9,
3,
4,
2,
5,
2,
6,
1,
8,
6,
1,
21,
3,