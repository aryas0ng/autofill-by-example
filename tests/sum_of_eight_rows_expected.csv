11,85
15,79
2,70
5,77
3,75
2,76
4,76
43,77
5,36
6,37
9,32
3,31
4,34
2,31
5,50
2,48
6,46
1,40
8,39
6,31
1,25
21,24
3,3